docs/src/_data/sites/zh-hans.yml

```
@@ -0,0 +1,116 @@
+#------------------------------------------------------------------------------
+# Simplified Chinese Site Details
+# The documentation site that is hosted at zh-hans.eslint.org/docs
+# Author: Percy Ma
+#------------------------------------------------------------------------------
+
+#------------------------------------------------------------------------------
+# Global Settings
+#------------------------------------------------------------------------------
+
+language:
+  code: zh-hans
+  flag: 🇨🇳
+  name: 简体中文
+locale: zh-hans
+hostname: zh-hans.eslint.org
+
+#------------------------------------------------------------------------------
+# Analytics
+#------------------------------------------------------------------------------
+
+google_analytics:
+  code: ""
```

```suggestion
  code: "G-6ELXTK7GZR"
```

I think we can use the same google analytics code here

https://github.com/eslint/eslint.org/blob/86f715bd427ffa23619aa3f5a90c507f8d8eb8ff/src/_data/sites/zh-hans.yml#L29